models/convert.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2019 The Gitea Authors. All rights reserved.
```
Small nit, only mentioning because I was already reviewing. 😂